src/css/wizard-navigation-bar.scss

```diff
@@ -16,6 +16,11 @@ $aw-param-indicator-state: null;
 // wizard step state: 'default', 'current', 'done' etc.
 $aw-param-step-state: null;
 
+@mixin rtl {
+  @at-root [dir="rtl"] #{&} {
+    @content;
+  }
```

Comment:
I'm not that firm with `scss` therefore my question:

To which element must `dir="rtl"` be applied to `aw-wizard` or something above it?

Reply:
aw-wizard is the lowest element that if dir="rtl" is applied to it will make it look properly for rtl because it's the container of the archwizard component so it will affect everything inside it.
In typical rtl web applications, the dir="rtl" is commonly applied to the HTML element tag or the body element tag. to make everything on the page get affected by the rtl direction.
I hope I could clear this point up to you.